CHANGES

```diff
@@ -1,6 +1,111 @@
 This file describes the most significant changes. For more detail, use
 'git log' on a clone of the charm repository.
 
+================================================================================
+What's new in Charm++ 6.11.0
+================================================================================
+
+This is a feature release, with the following major changes:
+
+Misc:
+
+- Charm++ now builds with CMake by default when using the `./build`
+  script (requires version 3.4 or higher). The old build system is
+  still available by using `./buildold`. Please see
+  https://charm.readthedocs.io/en/latest/charm++/manual.html#installation-with-cmake
+  for more information.
+
+- We have adopted GitHub discussions
+  (https://github.com/UIUC-PPL/charm/discussions) as the preferred
+  venue for Charm++ questions and discussions instead of the
+  charm@cs.illinois.edu mailing list.
+
+- The BigSim emulation facility has been removed from the runtime
+  system.
```

Comment:
```suggestion
- The BigSim emulator has been removed from Charm++.
```

Reply:
Is there any other simulator that we can recommend as a substitute?

Reply:
Potentially https://computing.llnl.gov/projects/interconnection-networks/network-modeling-and-simulation, but I'm not sure if that's really a complete replacement.